engines/stark/ui/cursor.cpp

```diff
@@ -74,7 +74,7 @@ void Cursor::setCursorImage(VisualImageXMG *image) {
 
 
 void Cursor::setMousePosition(const Common::Point &pos) {
-	_mousePos = _gfx->getScreenPosBounded(pos);
```

Comment:
Ok, now `getScreenPosBounded` is not used anymore and can be removed.

Also this does not make sure the cursor is set to inactive when over the black borders. For example, in the TLJ demo, when hovering the quit button in the top bar and moving the mouse to the right over the black border, the cursor stays active.

Reply:
Would adding a function `Gfx::Driver::isPosInScreenBounds` be fine? I could probably use that to do some checks for borders, maybe in `Stark::Cursor::Render`

Reply:
Ok, sounds fine. Did you notice `Driver::getScreenViewport`?

Reply:
Yeah, shall I use that for bound checking?

Reply:
You might want to.